plugins/node/opentelemetry-instrumentation-oracledb/src/internal-types.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright (c) 2025, Oracle and/or its affiliates.
+ * */
+
+import * as oracledbTypes from 'oracledb';
+import type * as api from '@opentelemetry/api';
+import { SpanConnectionConfig } from './types';
+
+// onEnterFn returns this Context(contains only span for now) and it is
+// received in onExitFn to end the span.
+export interface InstrumentationContext {
+  span: api.Span;
+}
+
+// Captures the entire span data
```

Comment:
you can add a short comment documenting that this interface correlates to a js object that is exported from the instrumented package. preferably with a link

Reply:
Added following:
```

// This corresponds to js object filled by the 'oracledb' module
// See: https://github.com/oracle/node-oracledb/blob/main/lib/traceHandler.js
```